Sources/SwiftDocC/Model/Rendering/RenderNode/RenderMetadata.swift

```diff
@@ -77,6 +77,8 @@ public struct RenderMetadata: VariantContainer {
     /// Authors can use the `@PageImage` metadata directive to provide custom icon and card images for a page.
     public var images: [TopicImage] = []
     
+    public var customMetadata: [String: String] = [:]
```

Comment:
Maybe we can use the doc comment you had in the `TopicRenderReference` here?

```suggestion
    /// Author provided custom metadata describing this page.
    public var customMetadata: [String: String] = [:]
```

Reply:
@ethan-kusters done.